packages/app/src/app/pages/CLI/Prompt/index.tsx

```diff
@@ -45,13 +38,15 @@ export const Prompt: React.FC<IPromptProps> = ({
         </SubTitle>
 
         <Buttons>
-          <Button onClick={signIn}>Sign in with GitHub</Button>
+          <Button onClick={() => signInCliClicked()}>
+            Sign in with GitHub
+          </Button>
         </Buttons>
       </Container>
     );
   }
 
-  if (loading) {
+  if (isLoadingCLI) {
```

Comment:
Where did this come from?

Was it changed in the state?

Reply:
This was first passed to `Prompt` as a prop by `CLI`, but can just be taken from the `useOvermind` hook

Reply:
OHH

Cool! Will merge :)